schema_salad/ref_resolver.py

```diff
@@ -1045,8 +1045,11 @@ def validate_links(self, document, base_url, all_doc_ids):
                     if identifier in document:
                         sl = SourceLine(document, identifier, validate.ValidationException)
                         if document[identifier] in all_doc_ids and sl.makeLead() != all_doc_ids[document[identifier]]:
-                            raise validate.ValidationException(
-                                "%s object %s `%s` previously defined" % (all_doc_ids[document[identifier]], identifier, relname(document[identifier]), ))
```

Comment:
Lets log this at the INFO or WARNING level instead of completely removing it.

Reply:
Sounds good. I'll push some code for that. Thank you.